src/core/cgroup.c

```diff
@@ -2996,6 +2996,127 @@ int unit_get_ip_accounting(
         return r;
 }
 
+static int unit_get_io_accounting_raw(Unit *u, uint64_t ret[static _CGROUP_IO_ACCOUNTING_METRIC_MAX]) {
+        static const char *const field_names[_CGROUP_IO_ACCOUNTING_METRIC_MAX] = {
+                [CGROUP_IO_READ_BYTES]       = "rbytes=",
+                [CGROUP_IO_WRITE_BYTES]      = "wbytes=",
+                [CGROUP_IO_READ_OPERATIONS]  = "rios=",
+                [CGROUP_IO_WRITE_OPERATIONS] = "wios=",
```

Comment:
IO_READ is really I == INPUT, and IO_WRITE is really O == OUTPUT. Wouldn't "reads=", "writes=" be better here?

Reply:
Or "rops=" and "wops=", which seems more consistent to me with "rbytes=" and "wbytes=".  /bikeshed

Reply:
these fields are exposed like that under those names from the kernel. This is not our naming but the kernel's own, we can just submit to their nomenclature on this and have no other option...

Or do I misunderstand what you are trying to say?

(The kernel block IO layer usually operates in terms of read/write rather than input/output when it comes to disk accesses, since it makes clearer which direction the data flows given the layer's position in the middle of everything, and for i/o the pov has an effect for r/w not so much...)

Reply:
I'm not talking about the kernel naming, but about our naming, i.e. the RHS in the assignments in this table.

"IO_READ_BYTES" → "rbytes", OK.
"IO_READ_OPERATIONS" → this should be "roperations", to be fully consistent. I suggests "reads", @vcaputo suggested "rops". I think either is better than "rios". ("rios" is short for "read input output s" which just doesn't make much sense ;)).

Reply:
Again: these names are not chosen by us. They are the field names the kernel exposes this under and if we want to parse the kernels files correctly we need to know what to look for.